src/log/MonologTarget.php

```diff
@@ -264,6 +285,11 @@ public function setFormatter(?FormatterInterface $formatter): void
         $this->_setLoggerProperty('formatter', $formatter);
     }
 
+    public function getFormatter(): ?FormatterInterface
+    {
+        return $this->formatter;
+    }
```

Comment:
This should be gettable so you can set the configured formatter when setting your handler.